packages/angular/projects/clr-angular/src/data/datagrid/datagrid-row.ts

```diff
@@ -132,13 +132,13 @@ export class ClrDatagridRow<T = any> implements AfterContentInit, AfterViewInit
     }
   }
 
-  // By default every item is selectable
+  // By default every item is selectable; it becomes not selectable only if it's explicitly set to false
   @Input('clrDgSelectable')
   public set clrDgSelectable(value: boolean) {
     this.selection.lockItem(this.item, value === false);
   }
 
-  public get clrDgSelectable() {
+  public get isSelectable() {
```

Comment:
I don't think we should change this public property name. It's basically a breaking change.

Reply:
This is a super internal API that is really hard to access and if it's not used inside the template I will prefer to be private. You as a developer should not have access to it and use the `Selection` and another service to change the state. But this is a limitation of  TypeScript and Angular integration when it comes to accessing variables inside the template (or linter setting)

It's improving a lot the readability when it comes to reading the template (the only place where this is used). Now is clear that if this is `true` then the `ngIf` will be displayed. With the strange behavior of `clrDgSelectable` that was not clear at first. 

So I think that this is a minor unnoticeable change that could improve the code at first. (It's a bad design on my part a year ago naming it like that)

Reply:
No, how do you know it's super internal API? If there are teams and users already making use of this property, we will break their app once they upgrade to our next release. We even prefixed the property with `clr` to indicate it's a public API property.